Common/include/option_structure.hpp

```diff
@@ -174,34 +147,35 @@ enum AVERAGE_TYPE {
   AVERAGE_AREA = 1,     /*!< \brief Area-weighted average. */
```

Comment:
While major changes are happening in this file, it could be a good time to finally remove the integer specifiers for each value in the enums, which aren't needed. Up to you if you have time

Reply:
Hmmm If we did not cast every enum to unsigned short in CConfig I would agree, and it should be doable with a script. But for debugging I find it useful to know what number to expect without having to count / print extra stuff.